tests/blackbox/stratis_cli_cert.py

```diff
@@ -175,12 +220,24 @@ def test_pool_list_empty(self):
         """
         self.unittest_command([_STRATIS_CLI, "pool", "list"], 0, True, False)
 
+    def test_pool_list_empty_permissions(self):
```

Comment:
Much better for pool and filesystem to test on a non-empty list. All the interesting action happens when there is a pool, because that's when we need to use the multiiple different interfaces, FetchProperties, and whatever else, to get the various chunks of information.